packages/node/src/init.ts

```diff
@@ -0,0 +1,9 @@
+import { init } from './sdk/init';
+
+/**
+ * The @sentry/node/init export can be used with the node --import and --require args to initialize the SDK entirely via
+ * environment variables.
+ *
+ * > SENTRY_DSN=https://examplePublicKey@o0.ingest.sentry.io/0 node --import=@sentry/node/init app.mjs
```

Comment:
let's add `SENTRY_TRACES_SAMPLE_RATE=1.0` to the snippet here to encourage having performance monitoring enabled.